src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -160,7 +160,7 @@ private ViewModel buildGapicClientViewModel(GapicInterfaceContext context) {
     apiImplClass.protoFilename(context.getInterface().getFile().getSimpleName());
     String implName = namer.getApiWrapperClassImplName(context.getInterfaceConfig());
     apiImplClass.name(implName);
-    String serviceAddress = context.getModel().getServiceConfig().getName();
+    String serviceAddress = context.getServiceAddress();
```

Comment:
This change is to follow the convention set by https://github.com/googleapis/gapic-generator/pull/2328